packages/web/src/scss/components/Modal/_Modal.scss

```diff
@@ -69,3 +69,17 @@
     justify-content: flex-end;
     margin-bottom: theme.$header-spacing;
 }
+
+.Modal__footer {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    margin-top: theme.$footer-spacing;
+    gap: theme.$gap-spacing;
+}
+
+.Modal__close {
```

Comment:
this is not used anywhere

Reply:
It is used on React version, because the align of Header is based on end, but when i want to use some title, also title will be aligned to end along with close button. So in React version is close button separated from header and aligned to Dialog element.

Reply:
Well, I think that if we extend this functionality in React we must also extend it to the web. So please extend also the demo example or create and issue where you describe that we must do this also for the web package.

Reply:
I've tested this locally. Do not know what messed up that it is not deployed.